src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj

```diff
@@ -2,14 +2,11 @@
   <Import Project=".\Common.GenevaExporter.props"/>
 
   <PropertyGroup>
+    <TargetFrameworks>$(TargetFrameworksForLibraries)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Description>An OpenTelemetry .NET exporter that exports to local ETW or UDS</Description>
-    <Authors>OpenTelemetry Authors</Authors>
     <!-- Tweak style rules for Geneva: Allow underscores in constant names and allow regions inside code blocks -->
     <NoWarn>$(NoWarn);SA1123;SA1310</NoWarn>
-    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>net8.0;net6.0;netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net462</TargetFrameworks>
```

Comment:
Where did the OS check go?

Reply:
Do you think this check is necessary? Looks like the CI is green also on linux so not sure if we should keep it. Otherwise, I can reinclude it in the Common.props.